libs/brackets-server/embedded-ext/wasm-debug/nls/ko/strings.js

```diff
@@ -0,0 +1,6 @@
+// Korean - ko strings
+define({
+    "ERROR_TRANSLATE_WASM"    : "WASM을 변환하는데 실패했습니다!",
+    "ERROR_NOT_FOUND_FILE"    : "선택된 파일이 없습니다.",
```

Comment:
may be, ERROR_NOT_FOUND_FILE -> ERROR_FILE_NOT_FOUND looks better ?

Reply:
Fixed.